docs/motivation.md

```diff
@@ -19,8 +19,7 @@ Most of the time you’ll be using either AutoForm or ValidatedForm, but there a
 > -- <cite>[Radosław Miernik](https://github.com/radekmie)</cite>
 
 <p align="center">
-  <img src="/img/hoc-hell.png" />
-  <small style={{display: 'block'}}>Screenshot of React DevTools when lots of HOC’s are applied to a component.</small>
+  <img src="/img/hoc-hell.png" alt="React DevTools when lots of HOC’s are applied to a component." />
```

Comment:
I honestly do not know what to do with this. I'm not a big fan of this image/example in the first place as to me it is lacking some context describing what causes such HoC hell to happen. But to not overcomplicate things I think we can just let this stay, maybe mention how many HoCs are actually rendered for easier comparison.